frontend/src/components/landing/BundleBanner.tsx

```diff
@@ -60,8 +60,8 @@ export const BundleBanner = (props: Props) => {
   const mainImage = (
     <img
       src={womanInBanner400w.src}
-      srcSet={`${womanInBanner400w.src} 400w, ${womanInBanner768w.src} 768w, ${womanInBanner1280w.src} 1280w`}
-      sizes={`(max-width: 600px) 400px, 768px, 1280px`}
+      srcSet={`${womanInBanner768w.src} 768w, ${womanInBanner400w.src} 400w`}
```

Comment:
Dropped the 1280w banner all together. Quality wise, it's not noticeable, and it was by far the largest image loaded on the page with the revisions.